cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/ReplicatorBuilder.java

```diff
@@ -44,6 +44,14 @@
      */
     public static class Push extends ReplicatorBuilder<Datastore, URI, Push> {
 
+        private int changeLimitPerBatch = 500;
+
+        private int batchLimitPerRun = 100;
+
+        private int bulkInsertSize = 10;
+
```

Comment:
I think they should be pushed up to the super class (leaving `attachmentsInline`), along with the options for pull, the constructor can set the defaults. And the Builder methods can be on the super class then too